mk/k3d.mk

```diff
@@ -44,6 +44,7 @@ K3D_CLUSTER_CREATE_OPTS ?= -i rancher/k3s:$(CI_K3S_VERSION) \
 	--k3s-arg '--no-deploy=traefik@server:0' \
 	--k3s-arg '--disable=metrics-server@server:0' \
 	--network kind \
+	--volume /sys/fs/bpf:/sys/fs/bpf:shared \
```

Comment:
Would I still be able to run this on my MacOS with Docker Desktop? I think it has problems with `shared` mounts.

Reply:
You might be right, I need to give it a try and check. I think in this case only kind can work :(

Reply:
we can parametrize to only mount this volume when OS is Linux

Reply:
I've added this in the newest changes but I encountered another problem with K3d and CircleCI which I didn't have on my VM. ```warning: found mount /sys/fs/cgroup, but it's type (tmpfs) is not cgroup2 - ignoring Error: failed to setup transparent proxy: getting cgroup failed with error: cannot find any cgroup2 mounts``` looking how to fix this